ibm/resource_ibm_dl_gateway.go

```diff
@@ -84,6 +85,12 @@ func resourceIBMDLGateway() *schema.Resource {
 		),
 
 		Schema: map[string]*schema.Schema{
+			dlAuthenticationKey: {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
same add sensitive : true

Reply:
This authentication key displays the CRN of the key rather than the key secret. So it is okay to show this information